src/Compositor.cpp

```diff
@@ -141,6 +146,12 @@ CCompositor::CCompositor() {
 
     m_sWLRToplevelMgr = wlr_foreign_toplevel_manager_v1_create(m_sWLDisplay);
 
+    m_sWRLDRMLeaseMgr = wlr_drm_lease_v1_manager_create(m_sWLDisplay, m_sWLRBackend);
+	if (!m_sWRLDRMLeaseMgr) {
+        Debug::log(CRIT, "Failed to create wlr_drm_lease_v1_manager");
```

Comment:
it's not critical, we don't exit

Reply:
that's my bad there was an exit but DRM leasing doesn't work when Hyprland is nested so I forgot to change the log level when i removed the exit. fixed.